qutip/core/qobjevo.py

```diff
@@ -1558,7 +835,7 @@ def _dynamic_argument_raise(op, state):
     )
 
 
-def _dynamic_argument(t, op, state, what, e_op):
+def dynamic_argument(t, op, state, what, e_op):
```

Comment:
We should be careful we don't accidentally export `dynamic_argument` into the `qutip` namespace, but I think we're ok just omitting it from `__all__`.